packages/pubsub/lib/utils/serde.ts

```diff
@@ -0,0 +1,23 @@
+import { serialize, deserialize } from 'v8';
```

Comment:
```suggestion
import { serialize, deserialize } from 'node:v8';
```

There is a package named v8, it's a bit confusing